Applications/Hermes/Readme.md

```diff
@@ -0,0 +1,105 @@
+# Hermes - Message Application
+
+Hermes is a simple messaging application developed for Faraday. Hermes was the God of messengers so it seemed fitting.
+
+The messaging functionality is the result of a Flask server that allows Transmit/Receive capability for locally connected Faraday devices. When the Flask server is running, requests sent to the specific port of operation allow the transmission and reception of data messages wirelessly between two (or more) Faraday devices.
```

Comment:
Based on Flask like many other Faraday applications, Hermes provides transmit and receive messaging functionality for Faraday radios within range of each other. Sending a message to the Flask port with an HTTP request will cause Hermes to interface with the local Faraday radio via Proxy to send the message over RF. The reverse is also true for receive.